db/connect.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+
+// http://mongoosejs.com/docs/promises.html
+mongoose.Promise = global.Promise
```

Comment:
I'm not like, 100% fluent in promises or whatever, but I understand it might be good to use bluebird?

> While mpromise is sufficient for basic use cases, advanced users may want to plug in their favorite ES6-style promises library like bluebird, or just use native ES6 promises. Just set mongoose.Promise to your favorite ES6-style promise constructor and mongoose will use it.

Although are we jsut gonna be a basic use case